ExAC.pm

```diff
@@ -256,16 +255,25 @@ sub run {
             foreach my $a(@vcf_alleles) {
               my $ac = shift @ac;
               $an = shift @an if @an;
-
+              my $matched_allele_string;
+              my $trimmed_seq_hash = trim_sequences($ref_allele, $a);
```

Comment:
Maybe overkilling, mainly because the vast majority of variants has only 1 alternate allele, but would it be more efficient to trim the `$ref_allele` outside the "for" loop ?

Reply:
We can't, we can only trim the two alleles relative to each other. The bug that this PR is attempting to fix is for multiple alternate alleles, where frequency data for the wrong allele is pulled out due to the allele strings not being trimmed.

Reply:
Ah ok fair enough. I didn't look closely at what trim_sequences was doing